wordpress-mobile/WordPress-Android
# Remove Domain Registration feature flag

This PR can be reviewed after #10135 is merged.

Fixes #9815

This PR removes feature flag for domain registration.

To test:

- install Vanila build.
- Select a site on a premium plan with domain credit.
- Notice that domain registration CTA appears at MySite fragment.
- Profit!

Update release notes:
